### PR TITLE
Fix for ISSUE-30 we make it possible to use the billing address as shipToAddress in case no shipping address exists on the order

### PR DESCRIPTION
**Pull request by duckchip**
When an order is placed without a shipping address (in case of purchasing only downloadable products), the order can't be send to ls Retail because the module depends on the shipping address. This PR let us use the billing address in case of a missing shipping Address. 

### Fixed Issues (if relevant)

1. lsretailomni/lsmag-two#30

### Manual testing scenarios (*)
1. Buy a downloadable product (which don't have a shipping address) for example a gift card
2. Try to sync the order to ls Retail in the back-office on the order detail page
3. Order is successfully synced

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by lsretailomni** [comment]
@duckchip thanks for working on the issue. PR has been merged into master and will be published in our next release.

File: src/Omni/Helper/OrderHelper.php [hunk 1]
```diff
@@ -117,21 +117,31 @@ public function prepareOrder(Model\Order $order, $oneListCalculateResponse)
             $storeId       = $oneListCalculateResponse->getStoreId();
             $cardId        = $oneListCalculateResponse->getCardId();
             $customerEmail = $order->getCustomerEmail();
-            $customerName  = $order->getShippingAddress()->getFirstname() .
-                ' ' . $order->getShippingAddress()->getLastname();
+            $customerName  = $order->getBillingAddress()->getFirstname() .
++                ' ' . $order->getBillingAddress()->getLastname();
```
Comment (hostep) [inline src/Omni/Helper/OrderHelper.php:121]: This extra `+` character might introduce problems, no?

https://github.com/lsretailomni/lsmag-two/blob/11d71db949b6c35d1f5d1160463fe915df8abbf0/src/Omni/Helper/OrderHelper.php#L121

@duckchip, @lsretailomni

Comment (duckchip) [reply src/Omni/Helper/OrderHelper.php:121]: good catch. This will introduce errors. But this branch is already merged. @lsretailomni

**Comment by lsretailomni** [comment]
@hostep thanks for reporting, the team has already fixed this in the master.

**Pull request merged**